### PR TITLE
Seedlet: Update theme to work with the new marketing bar

### PR DESCRIPTION
**Pull request by MaggieCabrera**
#### Changes proposed in this Pull Request:

Updated styles on Seedlet to fit with the new marketing bar.

#### How to test:

* Update your sandbox with the contents of this PR.
* Modify the following file on your sandbox:
`https://opengrok.a8c.com/source/xref/trunk/wp-content/blog-plugins/marketing-bar.php`
* If your site is older than 2018-05-16 change this line:
`const SHOW_MARKETING_BAR_FOR_BLOGS_REGISTERED_ON_OR_AFTER = '2018-05-16';`
* If modulo of your user id is not 7, change the `!===` to `==`
`if ( is_user_logged_in() && 7 !== (int) $blog_owner % 10 ) {`

#### Screenshots:

Seedlet, with wp admin bar:

Seedlet, without wp admin bar:

#### Related issue(s):

#2629

### CODE REVIEW

File: seedlet/assets/sass/components/header/_config.scss [hunk 1]
```diff
@@ -33,4 +33,7 @@
 	--social-nav--color-link: var(--global--color-foreground);
 	--social-nav--color-link-hover: var(--global--color-primary-hover);
 	--social-nav--padding: calc( 0.5 * var(--primary-nav--padding) );
+
+	--marketing-bar--height: 45px;
```
Comment (jffng) [inline seedlet/assets/sass/components/header/_config.scss:37]: How about renaming this to be context specific? 

 ```suggestion
	--wpcom--marketing-bar--height: 45px;
```

File: seedlet/assets/sass/components/header/_config.scss [hunk 2]
```diff
@@ -33,4 +33,7 @@
...
+	--wpadmin-bar--height: 46px;
```
Comment (jffng) [inline seedlet/assets/sass/components/header/_config.scss:38]: I'm pretty sure we use this value in the primary nav too so would be good to update there as a follow-up.

**Review by jffng — COMMENTED** [review]
I can't get the marketing bar to show up on my sandboxed site... but I left a few general comments :)

**Comment by scruffian** [comment]
I tried a different approach in #2635

**Review by scruffian — APPROVED** [review]
LGTM

**Pull request merged**